qemu/target-i386/translate.c

```diff
@@ -6110,6 +6115,7 @@ static target_ulong disas_insn(CPUX86State *env, DisasContext *s,
                         /* fcomp needs pop */
                         gen_helper_fpop(tcg_ctx, cpu_env);
                     }
+                    fpu_update_ip(env, pc_start);
```

Comment:
This fix is wrong according to the discussion at https://lists.nongnu.org/archive/html/qemu-devel/2021-04/msg00307.html.

I have submitted a patch at https://lists.nongnu.org/archive/html/qemu-devel/2021-04/msg02868.html.

Just a ping and reminder for this issue.